vcpkg.json

```diff
@@ -31,7 +31,10 @@
             "wayland"
           ],
           "platform": "linux"
-        }
+        },
+        "libcurl-simple-https",
+        "libarchive",
+        "nlohmann-json"
       ]
```

Comment:
Nlohmann is not available on the Steam runtime, so please add it in the list below. I haven't checked, but I think you can add it to the flatpak list too since it's probably missing there too.

Reply:
done